src/rpc/rawtransaction.cpp

```diff
@@ -610,7 +610,7 @@ static UniValue combinerawtransaction(const JSONRPCRequest& request)
             "\"hex\"            (string) The hex-encoded raw transaction with signature(s)\n"
                 },
                 RPCExamples{
-                    HelpExampleCli("combinerawtransaction", "[\"myhex1\", \"myhex2\", \"myhex3\"]")
+                    HelpExampleCli("combinerawtransaction", "\"[\\\"myhex1\\\", \\\"myhex2\\\", \\\"myhex3\\\"]\"")
```

Comment:
why not use single quotes here? that'd avoid having to escape the inner `"`'s

Reply:
Done.